docs/copied-from-beats/outputconfig.asciidoc

```diff
@@ -219,8 +219,12 @@ The index name to write events to. The default is
 need to configure the `setup.template.name` and `setup.template.pattern` options
 (see <<configuration-template>>).
 
+////
```

Comment:
Ahh, I was wrong. This line remains, but we've added a `no_dashboards` attribute to hide it. I'll add the attribute the `index.asciidoc` file and commit the changes.